src/main.py

```diff
@@ -128,7 +128,155 @@ def embed_text(text):
     print("Done.")
 
     print("Creating tensorflow session...")
-    session = tf.Session()
+    config = tf.ConfigProto()
+    config.gpu_options.per_process_gpu_memory_fraction = 0.5
+    session = tf.Session(config=config)
+    session.run(tf.global_variables_initializer())
+    session.run(tf.tables_initializer())
+    print("Done.")
+
+    client = Elasticsearch()
```

Comment:
I don't think the script needs to be duplicated here.

Reply:
I don't know why it looks like a duplication. I just added 3 lines here 

https://github.com/svola/text-embeddings/blob/limit_memory/src/main.py#L277-L279

and the config option:

https://github.com/svola/text-embeddings/blob/limit_memory/src/main.py#L268

Reply:
Hello again, the script has been duplicated in the file you linked to. Here is the first line where the script begins to repeat itself: https://github.com/svola/text-embeddings/blob/limit_memory/src/main.py#L146

Reply:
Oh so sorry. That was a copy-paste error. It's fixed now.